tasks/setup.py

```diff
@@ -0,0 +1,66 @@
+import os
+import os.path
+from pathlib import Path
+import getpass
+
+from invoke import task
+from invoke.context import Context
+
+from .config import Config, profile_filename
+from .tool import ask, debug, info, warn
+
+@task
+def setup(ctx: Context):
+    """
+    Setup a local environment interactively
+    """
+    info("🤖 Install Pulumi")
+    os.system("brew install pulumi/tap/pulumi")
+    os.system("pulumi login --local")
+
+    info("🤖 Let's configure your environment for e2e tests! Press ctrl+c to stop me")
+    config = Config()
+    config.configParams = Config.Params()
+    config.configParams.agent = Config.Params.Agent()
+    config.configParams.aws = Config.Params.Aws()
+
+    config.configParams.aws.keyPairName = getpass.getuser()
+    keyPairName = ask(f"🔑 Key pair name - stored in AWS Sandbox, default [{config.configParams.aws.keyPairName}]: ")
+    if len(keyPairName) > 0:
+        config.configParams.aws.keyPairName = keyPairName
+
+    checkKeyPair = ask("Did you create your SSH key on AWS and want me to check it is loaded on your ssh agent when creating manual environments or running e2e tests [Y/N]? Default No: ")
+    if len(checkKeyPair) > 0:
+        config.options = Config.Options()
+        config.options.checkKeyPair = checkKeyPair.lower() == "y" or checkKeyPair.lower() == "yes"
+
+    config.configParams.aws.publicKeyPath = Path.home().joinpath(".ssh", "id_ed25519.pub")
```

Comment:
Is `id_ed25519.pub` really the default public key name?

Reply:
To enhance compatibility you should probably fallback to `id_rsa.pub` for people (majority) that onboarded before this change.